apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -110,18 +114,47 @@ export default function Highlights( { siteId, gmtOffset, odysseyStatsBaseUrl } )
 		queryDate
 	);
 
+	const headingTitle = translate( '7 Day Highlights' );
+	const topPostsAndPagesTitle = translate( 'Top Posts & Pages' );
+	const topReferrersTitle = translate( 'Top Referrers' );
+
+	const moduleTabs = [
+		{
+			value: 'topPostsAndPages',
+			label: topPostsAndPagesTitle,
+		},
+		{
+			value: 'topReferrers',
+			label: topReferrersTitle,
+		},
+	];
+
 	return (
-		<div
-			className="stats-widget-highlights stats-widget-card"
-			aria-label={ translate( '7 Day Highlights' ) }
-		>
+		<div className="stats-widget-highlights stats-widget-card" aria-label={ headingTitle }>
 			<div className="stats-widget-highlights__header">
-				<label>{ translate( '7 Day Highlights' ) }</label>
+				<label>{ headingTitle }</label>
+			</div>
+			<div className="stats-widget-highlights__tabs">
+				<SegmentedControl primary className="test">
+					{ moduleTabs.map( ( tab ) => {
+						return (
+							<SegmentedControl.Item
+								key={ tab.value }
+								selected={ tab.value === selectedTab }
+								onClick={ () => setSelectedTab( tab.value ) }
+							>
+								{ tab.label }
+							</SegmentedControl.Item>
+						);
+					} ) }
+				</SegmentedControl>
 			</div>
 			<div className="stats-widget-highlights__body">
 				<TopColumn
-					className="stats-widget-highlights__column"
-					title={ translate( 'Top Posts & Pages' ) }
+					className={ classNames( 'stats-widget-highlights__column', {
+						'is-mobile-show': selectedTab === 'topPostsAndPages',
```

Comment:
```suggestion
						'stats-widget-highlights__column--show-in-mobile': selectedTab === 'topPostsAndPages',
```

Nit: Recommending BEM class name here to avoid any styles leaking onto our columns.

Reply:
Addressed in 96688611eb